gax-java/gax/src/test/java/com/google/api/gax/rpc/PagingTest.java

```diff
@@ -133,6 +133,54 @@ public void pagedByPage() {
     Truth.assertThat(requestCapture.getAllValues()).containsExactly(0, 2, 4).inOrder();
   }
 
+  @Test
+  public void streamedByPage() {
```

Comment:
Can we update the test names based on the [best practices](https://engdoc.corp.google.com/eng/doc/devguide/testing/unit/best-practices.md?cl=head&polyglot=java#naming) and a recent [TotT](https://engdoc.corp.google.com/eng/doc/tott/episodes/660.md?cl=head)?

Reply:
done